### ISSUE TITLE
Key combinations don't repeat when held

### ISSUE THREAD

**Issue by Riteo** [Orama-Interactive/Pixelorama#401]
**Pixelorama version:**
v0.8.2-stable

**OS/device including version:**
Arch Linux 5.9.11.arch2 i3/Nvidia 1060 3GB drivers version 455.45.01

**Issue description:**
While holding ctrl+z, ctrl+y or any other key combination, their respective action gets executed only once.
It get's tiring really fast to keep pressing the same combination over and over again.

**Steps to reproduce:**
Draw a bunch of lines, then hold ctrl+z. Notice how only the last action gets reverted, and nothing more.

If this isn't intended behaviour, I can send a PR myself.

**Comment by Schweini07** [comment]
I wouldn't Label this as a bug, rather a enhancement. The feature you are describing is just not implemented. 
If you have nothing against it @OverloadedOrama I'd like to try to tackle this myself.

**Comment by OverloadedOrama** [comment]
> I wouldn't Label this as a bug, rather a enhancement. The feature you are describing is just not implemented.
> If you have nothing against it @OverloadedOrama I'd like to try to tackle this myself.

@Schweini07 Sure, go for it!

**Comment by Schweini07** [comment]
I think got it to work.
![2020-12-16 13-11-29](https://user-images.githubusercontent.com/47503977/102347557-d7a27e80-3fa0-11eb-9afc-f47806407157.gif)
It's a pretty interesting implementation and I quite like it.
I just have to wait until my move tool PR is closed to open this PR, as I am not using an extra branch for the move tool, but master, because of "complications".
See you in hell Git.

**Comment by Riteo** [comment]
@Schweini07 one small thing: IMO those action notifications should be spaced a bit, but I guess that might be a separate issue.

**Comment by Schweini07** [comment]
I thought about that, but yes I think this should be something seperate.
